src/Kafka/ConsumerGroups/components/TopicSelect.tsx

```diff
@@ -0,0 +1,49 @@
+import { Select, SelectOption, SelectVariant } from "@patternfly/react-core";
+import type { SelectProps } from "@patternfly/react-core";
+import { useState } from "react";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+
+export type TopicSelectProps = {
+  value: string;
+  topics: string[];
+  onChange: (topic: string) => void;
+};
+
+export const TopicSelect: FunctionComponent<TopicSelectProps> = ({
+  topics,
+  onChange,
+  value,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const onToggle = (isOpen: boolean) => {
+    setIsOpen(isOpen);
+  };
+
+  const onSelect: SelectProps["onSelect"] = (_, topic) => {
+    onChange(topic as string);
+    setIsOpen(false);
+  };
+
+  return (
+    <Select
+      aira-label={t("consumerGroup.reset_offset_topic_label")}
+      toggleAriaLabel={t("consumerGroup.reset_offset_topic_label")}
+      id={"topic-select"}
+      variant={SelectVariant.single}
+      onToggle={onToggle}
+      onSelect={onSelect}
+      isOpen={isOpen}
+      selections={value}
+      placeholderText={t("common:select")}
```

Comment:
Is this the right placeholder? "Select" doesn't seem that informative to me.

Reply:
The same applies to all other cases where `common:select` is used.

Reply:
As per the design doc https://docs.google.com/presentation/d/1wAsmKnco9-ZzJadp1djCim_RqlBgWEumFi2Fbxd531Q/edit#slide=id.gdfee5733b1_6_1, the placeholder text is select

Reply:
@riccardo-forina, I have added the placeholder test as per the design https://docs.google.com/presentation/d/1wAsmKnco9-ZzJadp1djCim_RqlBgWEumFi2Fbxd531Q/edit#slide=id.gdfee5733b1_6_1. other than that I have address all other review comments